crnlib/crn_arealist.cpp

```diff
@@ -13,8 +13,8 @@ static void area_fatal_error(const char*, const char* pMsg, ...) {
   va_start(args, pMsg);
 
   char buf[512];
-#ifdef _MSC_VER
-  _vsnprintf_s(buf, sizeof(buf), pMsg, args);
+#if defined(_WIN32) || defined(_MSC_VER)

```

Comment:
Just check `_WIN32`. You can assume that `_MSC_VER` implies Windows.

Reply:
Done.